game/outfits.rpy

```diff
@@ -1273,6 +1255,12 @@ init -1 python in jn_outfits:
         unlocked=False,
         is_jn_wearable=True
     ))
+    __register_wearable(JNClothes(
+        reference_name="jn_clothes_QT_sweater",
+        display_name="QT sweater",
+        unlocked=False,
+        is_jn_wearable=True
+    ))
```

Comment:
I think this might be better if we make the reference name all lowercase to match the names of everything else - it'll just be a little more consistent.

Remember that we'd just need to make sure we rename the folder for the sweater assets as well!